Deploy/auto_boot.sh

```diff
@@ -7,6 +7,7 @@ cp $DIR_PATH/Systemd/ss-update-daemon.service /lib/systemd/system/ss-update-daem
 cp $DIR_PATH/Systemd/ss-susi-server.service /lib/systemd/system/ss-susi-server.service
 cp $DIR_PATH/Systemd/ss-python-flask.service /lib/systemd/system/ss-python-flask.service
 cp $DIR_PATH/Systemd/ss-susi-linux.service /lib/systemd/system/ss-susi-linux.service
+cp $DIR_PATH/Systemd/ss-susi-startup.service /lib/systemd/system/ss-susi-startup.service
```

Comment:
Hmm, I see that the file name is not _ss-susi-startup.service_.

Reply:
Sorry @hongquan . I'll update the PR